content/geometry/object-environment/environment-structure.adoc

```diff
@@ -0,0 +1,482 @@
+= Model structure
+
+== Structure overview
+
+[plantuml]
+----
+legend
+[[../geometry/object-classes.html#_environment_t Environment_Root (T)]]
+|_[[../geometry/object-classes.html#_terrain_t Terrain]]
+|_[[../geometry/object-classes.html#_environment_objects_t Objects]]
+  |_ [[../geometry/object-classes.html#_buildings Buildings]]
+  |_ [[../geometry/object-classes.html#_vegetation Vegetation]]
+  |_ [[../geometry/object-classes.html#_other_objects Other_Objects]]
+|_[[../geometry/object-classes.html#_road_network_t Road_Network]]
+  |_ [[../geometry/object-classes.html#_drivable_area_t Drivable_Area]]
+  |_ [[../geometry/object-classes.html#_sidewalk_t Sidewalk]]
+  |_ [[../geometry/object-classes.html#_road_marks_t Road_Marks]]
+  |_ [[../geometry/object-classes.html#_road_object_idx_t Road_Object_<object_idx> (T)]]
+  |_ [[../geometry/object-classes.html#_signal_signal_idx_t Signal_<signal_idx> (T)]]
+    |_ [[../geometry/object-classes.html#_sign_idx_t Sign_<sign_idx> (T)]]
+    |_ [[../geometry/object-classes.html#_light_idx_t Light_<light_idx> (T)]]
+    |_ [[../geometry/object-classes.html#_cover_idx_t Cover<cover_idx> (T)]]
+end legend
+----
+
+////
+[plantuml]
+----
+legend
+Root (T)
+|-- Terrain
+|-- Objects
+----|--Buildings
+----|--Vegetation
+|-- Road_Network
+----|--Driving_Area
+----|--Sidewalks
+----|--Road_Marks
+----|--Road_Object_<object_idx> (T)
+----|--Signal_<signal_idx> (T)
```

Comment:
Signal, Sign and Light are mentioned above, but here are other groups (Signal, Sign, Bulb) listed. Should be consistent with the structure above.